tests/build-conf/qemux86-64-bios-grub-gpt/bblayers.conf

```diff
@@ -11,4 +11,5 @@ BBLAYERS ?= " \
   @WORKSPACE@/meta-yocto-bsp \
   @WORKSPACE@/meta-mender/meta-mender-core \
   @WORKSPACE@/meta-mender/meta-mender-demo \
+  @WORKSPACE@/meta-openembedded/meta-oe \
```

Comment:
Not really related to this pull request, but just an FYI to @lluiscampos: I think integration-test-runner does not take the meta-openembedded layer into account when it matches PRs to meta-mender against poky branch. To illustrate with an example, if you submit a PR to meta-mender/master, you get this build:

```
META_MENDER_REV: pull/1234/head
POKY_REV: dunfell
META_OPENEMBEDDED_REV: dunfell
```

which is correct, but if you submit a PR to meta-mender/zeus, you get:

```
META_MENDER_REV: pull/1234/head
POKY_REV: zeus
META_OPENEMBEDDED_REV: dunfell
```

which is not correct. So essentially all automatic builds for non-master and non-dunfell branches (both of this use poky/dunfell as the their base) will break after this PR is merged, and have to be started manually.

Should we create a ticket for it?

Reply:
I am not sure about it, but will [this help](https://github.com/mendersoftware/integration-test-runner/pull/52)?

Reply:
Good catch, @kacf . I always set it up manually when running the pipeline for old yocto branches (even though only now I realize it was uselss). So indeed, we should get this in the bot.

I'll take care of this independently.